src/axes/axis.ts

```diff
@@ -218,6 +243,21 @@ export class Axis<D> extends Component {
     this._annotationWriter = new Typesettable.Writer(this._annotationMeasurer, typesetterContext);
   }
 
+  private _configureTickLabelContainer() {
+    if (this._tickLabelContainer != null) {
+      this._tickLabelContainer
+        // clearing to remove outdated font-size classes
+        .attr("class", null)
+        .classed(`${Axis.TICK_LABEL_CLASS}-container`, true)
+        .classed(`label-${this._tickLabelFontSize}`, true);
+    } else {
+      this._tickLabelContainer = this.content()
+        .append("g")
+        .classed(`${Axis.TICK_LABEL_CLASS}-container`, true)
+        .classed(`label-${this._tickLabelFontSize}`, true);
+    }
```

Comment:
Could we have the classes bits only once? 
```suggestion
    if (this._tickLabelContainer == null) {
      this._tickLabelContainer = this.content().append("g");
    }
    this._tickLabelContainer
      // clearing to remove outdated font-size classes
      .attr("class", null)
      .classed(`${Axis.TICK_LABEL_CLASS}-container`, true)
      .classed(`label-${this._tickLabelFontSize}`, true);
```